### PR TITLE
Change from custom Views to LX Views

### PR DESCRIPTION
**Pull request by jkbelcher**
Out with the old, in with the new!  For a thrilling 8 months we ran a custom implementation of pattern-level views.  Now the LX framework handles all this work.

This was a relatively quick pass at removing the old system.

Notes:
 - Because we're using a static model we still load the view definitions from text file and add them to the lx structure at startup.
 - I left a "View" knob in the UI on the right.  In LX now all channels & devices (patterns, effects) can have their own view.  Leaving this knob in the UI could be handy for pattern developers checking out their pattern on different views.
 
image

### CODE REVIEW

**Review by andrewlook — APPROVED** [review]
This is looking great! Thanks for the massive code cleanup.

I appreciate keeping the view per pattern knob enabled outside of the top 16 params, so that it's available in design mode to make it easier when developing patterns - definitely agree with the rationale there.

I tested this locally and everything seems to work well - I verified that the per-pattern view doesn't override any per-channel views but acts as a mask on the per-channel view. This is obviously an edge case, but it seemed logical to me how this is implemented 👍

File: src/main/java/titanicsend/model/TEWholeModel.java [hunk 1]
```diff
@@ -11,17 +11,20 @@
...
 import titanicsend.dmx.model.BeaconModel;
-import titanicsend.dmx.model.ChauvetSpot160Model;
 import titanicsend.dmx.model.DmxModel;
 import titanicsend.dmx.model.DmxWholeModel;
 import titanicsend.dmx.model.DmxModel.DmxCommonConfig;
 import titanicsend.lasercontrol.MovingTarget;
+import titanicsend.model.justin.ViewDefinition;
 import titanicsend.output.ChromatechSocket;
 import titanicsend.output.GrandShlomoStation;
+import titanicsend.pattern.jon.ModelBender;
```
[the remark this answers is not in the document]
Comment (jkbelcher) [reply src/main/java/titanicsend/model/TEWholeModel.java:27]: Fine by me.

[the remark this answers is not in the document]
Comment (jkbelcher) [reply src/main/java/titanicsend/model/TEWholeModel.java:26]: Moved ViewDefinition.

**Comment by jkbelcher** [comment]
@andrewlook I merged dev into this branch and moved the ViewDefinition file.  Ready for final review/merge.

**Review by andrewlook — APPROVED** [review]
Looks great! Thanks for making these final changes here 🙌

**Pull request merged**